<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>703</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>105</red>
        <green>84</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>105</red>
        <green>84</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>105</red>
        <green>84</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>105</red>
        <green>84</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Advanced Filters</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="spellsLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Stencil&quot;;</string>
    </property>
    <property name="text">
     <string>Spells</string>
    </property>
   </widget>
   <widget class="QLabel" name="coreStatsLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Stencil&quot;;</string>
    </property>
    <property name="text">
     <string>Core Stats</string>
    </property>
   </widget>
   <widget class="QLabel" name="proficienciesLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>460</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Stencil&quot;;</string>
    </property>
    <property name="text">
     <string>Further Proficiencies</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="coreStatsView">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>431</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(160, 160, 160);</string>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="renderHints">
     <set>QPainter::TextAntialiasing</set>
    </property>
   </widget>
   <widget class="QLabel" name="warningLabel1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>571</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Stencil&quot;;
color: rgb(131, 24, 14);</string>
    </property>
    <property name="text">
     <string>Please note, these filters are advanced and have heavy impact on</string>
    </property>
   </widget>
   <widget class="QLabel" name="warningLabel3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>611</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Stencil&quot;;
color: rgb(131, 24, 14);</string>
    </property>
    <property name="text">
     <string>Only use them if you have advanced knowledge of Dungeons &amp; Dragons.</string>
    </property>
   </widget>
   <widget class="QLabel" name="warningLabel2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Stencil&quot;;
color: rgb(131, 24, 14);</string>
    </property>
    <property name="text">
     <string>the results of the optimisation.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveAdvancedOptions">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>630</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 12pt &quot;Rockwell Extra Bold&quot;;
color: rgb(255, 255, 255);
background-color: rgb(28, 113, 2);
border: 2px solid #555;
border-radius: 20px;
border-style: outset;</string>
    </property>
    <property name="text">
     <string>Save Advanced Options</string>
    </property>
   </widget>
   <widget class="QComboBox" name="selectClass">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentText">
     <string>Select Class</string>
    </property>
    <item>
     <property name="text">
      <string>Select Class</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="selectSubclass">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentText">
     <string>Select Subclass</string>
    </property>
    <item>
     <property name="text">
      <string>Select Subclass</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="selectRace">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>140</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentText">
     <string>Select Race</string>
    </property>
    <item>
     <property name="text">
      <string>Select Race</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="selectSubrace">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>180</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentText">
     <string>Select Subrace</string>
    </property>
    <item>
     <property name="text">
      <string>Select Subrace</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="selectBackground">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="currentText">
     <string>Select Background</string>
    </property>
    <item>
     <property name="text">
      <string>Select Background</string>
     </property>
    </item>
   </widget>
   <widget class="QGraphicsView" name="spellsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>361</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(160, 160, 160);</string>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="renderHints">
     <set>QPainter::TextAntialiasing</set>
    </property>
   </widget>
   <widget class="QScrollArea" name="spellOptions">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 227, 227);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="spellContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>159</width>
       <height>99</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="spellSearchBar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter Spell Names</string>
    </property>
   </widget>
   <widget class="QPushButton" name="spellSearchBtn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Images/SearchImage.png</normaloff>../Images/SearchImage.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QScrollArea" name="chosenSpellOptions">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 227, 227);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="chosenSpellContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>159</width>
       <height>99</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="equipmentSearchBar">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>270</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter Equipment Names</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="equipmentView">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>250</y>
      <width>361</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(160, 160, 160);</string>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="renderHints">
     <set>QPainter::TextAntialiasing</set>
    </property>
   </widget>
   <widget class="QPushButton" name="equipmentSearchBtn">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>270</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Images/SearchImage.png</normaloff>../Images/SearchImage.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QGraphicsView" name="proficienciesView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>460</y>
      <width>721</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(160, 160, 160);</string>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="renderHints">
     <set>QPainter::TextAntialiasing</set>
    </property>
   </widget>
   <widget class="QLabel" name="armorLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>480</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Stencil&quot;;</string>
    </property>
    <property name="text">
     <string>Armor</string>
    </property>
   </widget>
   <widget class="QLabel" name="weaponsLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>480</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Stencil&quot;;</string>
    </property>
    <property name="text">
     <string>Weapons</string>
    </property>
   </widget>
   <widget class="QLabel" name="toolsLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>480</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Stencil&quot;;</string>
    </property>
    <property name="text">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QLabel" name="languagesLabel">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>480</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Stencil&quot;;</string>
    </property>
    <property name="text">
     <string>Languages</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="lightArmorBtn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>510</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Light Armor</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="mediumArmorBtn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>530</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Medium Armor</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="heavyArmorBtn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>550</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Heavy Armor</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="shieldBtn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>570</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Shields</string>
    </property>
   </widget>
   <widget class="QLabel" name="langAmntLabel">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>510</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="langAmntSpinner">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>510</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>7</number>
    </property>
   </widget>
   <widget class="QScrollArea" name="languagesScroller">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>540</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 227, 227);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>159</width>
       <height>69</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="toolsStack">
    <property name="geometry">
     <rect>
      <x>399</x>
      <y>500</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 227, 227);</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="QCheckBox" name="simpleWeaponsBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>510</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Simple Weapons</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="martialWeaponsBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>530</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Martial Weapons</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="weaponsMatchEquipmentBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>550</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Match Equipment Filters</string>
    </property>
   </widget>
   <widget class="QLabel" name="warningLabel4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Stencil&quot;;
color: rgb(131, 24, 14);</string>
    </property>
    <property name="text">
     <string>These filters are not checked for possibility.</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="chosenEquipmentOptions">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>330</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 227, 227);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="chosenEquipmentContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>159</width>
       <height>99</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="equipmentOptions">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>330</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 227, 227);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="equipmentContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>159</width>
       <height>99</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="equipmentLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>250</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Stencil&quot;;</string>
    </property>
    <property name="text">
     <string>Equipment</string>
    </property>
   </widget>
   <widget class="QLabel" name="spellSearchLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(152, 9, 152);
color: rgb(255, 255, 255);
font: 87 9pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>SEARCH RESULTS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="spellSelectedLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(7, 10, 181);
color: rgb(255, 255, 255);
font: 87 9pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>SELECTED</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="equipmentSearchLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>310</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(152, 9, 152);
color: rgb(255, 255, 255);
font: 87 9pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>SEARCH RESULTS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="equipmentSelectedLabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>310</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(7, 10, 181);
color: rgb(255, 255, 255);
font: 87 9pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>SELECTED</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>proficienciesView</zorder>
   <zorder>equipmentView</zorder>
   <zorder>spellsView</zorder>
   <zorder>coreStatsView</zorder>
   <zorder>spellsLabel</zorder>
   <zorder>coreStatsLabel</zorder>
   <zorder>proficienciesLabel</zorder>
   <zorder>warningLabel1</zorder>
   <zorder>warningLabel3</zorder>
   <zorder>warningLabel2</zorder>
   <zorder>saveAdvancedOptions</zorder>
   <zorder>selectClass</zorder>
   <zorder>selectSubclass</zorder>
   <zorder>selectRace</zorder>
   <zorder>selectSubrace</zorder>
   <zorder>selectBackground</zorder>
   <zorder>spellOptions</zorder>
   <zorder>spellSearchBar</zorder>
   <zorder>spellSearchBtn</zorder>
   <zorder>chosenSpellOptions</zorder>
   <zorder>equipmentSearchBar</zorder>
   <zorder>equipmentSearchBtn</zorder>
   <zorder>armorLabel</zorder>
   <zorder>weaponsLabel</zorder>
   <zorder>toolsLabel</zorder>
   <zorder>languagesLabel</zorder>
   <zorder>lightArmorBtn</zorder>
   <zorder>mediumArmorBtn</zorder>
   <zorder>heavyArmorBtn</zorder>
   <zorder>shieldBtn</zorder>
   <zorder>langAmntLabel</zorder>
   <zorder>langAmntSpinner</zorder>
   <zorder>languagesScroller</zorder>
   <zorder>toolsStack</zorder>
   <zorder>simpleWeaponsBtn</zorder>
   <zorder>martialWeaponsBtn</zorder>
   <zorder>weaponsMatchEquipmentBtn</zorder>
   <zorder>warningLabel4</zorder>
   <zorder>chosenEquipmentOptions</zorder>
   <zorder>equipmentOptions</zorder>
   <zorder>equipmentLabel</zorder>
   <zorder>spellSearchLabel</zorder>
   <zorder>spellSelectedLabel</zorder>
   <zorder>equipmentSearchLabel</zorder>
   <zorder>equipmentSelectedLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
